🌐 Add Traditional Chinese translation for `docs/zh-hant/docs/fastapi-people.md` (#11639)
fastapi/fastapi: docs/zh-hant/docs/fastapi-people.md

---
hide:
  - navigation
---

# FastAPI 社群

FastAPI 有一個非常棒的社群，歡迎來自不同背景的朋友參與。

## 作者

嘿! 👋

關於我:

{% if people %}
<div class="user-list user-list-center">
{% for user in people.maintainers %}

<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">解答問題: {{ user.answers }}</div><div class="count">Pull Requests: {{ user.prs }}</div></div>
{% endfor %}

</div>
{% endif %}

> 我是 **FastAPI** 的作者。你可以在[幫助 FastAPI - 獲得幫助 - 與作者聯繫](help-fastapi.md#connect-with-the-author){.internal-link target=_blank} 中閱讀更多相關資訊。

Reviewer: python doc 翻譯是習慣不留空，但的確選擇一個方式並統一作法即可 ([ref](https://github.com/sparanoid/chinese-copywriting-guidelines#%E8%B6%85%E9%80%A3%E7%B5%90%E4%B9%8B%E9%96%93%E5%A2%9E%E5%8A%A0%E7%A9%BA%E6%A0%BC))

> 我是 **FastAPI** 的作者。你可以在[幫助 FastAPI - 獲得幫助 - 與作者聯繫](help-fastapi.md#connect-with-the-author){.internal-link target=_blank} 中閱讀更多相關資訊。

Author: 感謝建議！
我這邊統一都不留空白，除非是英文字才會留空 :)

> ...但在這裡，我想向你介紹這個社群。

Reviewer 2: ```suggestion
——但在這裡，我想向你介紹這個社群。
```

Author: 感謝建議！
用 —— 好像也怪怪的，我這邊所以先保留跟原文一樣的寫法 :)

---

**FastAPI** 獲得了許多社群的大力支持。我想特別表揚他們的貢獻。

這些人包括：

> * [在 GitHub 上幫助他人解答問題](help-fastapi.md#help-others-with-questions-in-github){.internal-link target=_blank}。

Author: 我這邊調整成「在 GitHub "中"幫助他人解答問題」
```
* [在 GitHub 中幫助他人解答問題](help-fastapi.md#help-others-with-questions-in-github){.internal-link target=_blank}。
```

> * [創建 Pull Requests](help-fastapi.md#create-a-pull-request){.internal-link target=_blank}。

Reviewer: 創建是簡中慣用詞
```suggestion
* [建立 Pull Requests](help-fastapi.md#create-a-pull-request){.internal-link target=_blank}。
```

Author: 已調整

* 審查 Pull Requests，[尤其是翻譯方面的貢獻](contributing.md#translations){.internal-link target=_blank}。

讓我們為他們熱烈鼓掌。 👏 🙇

## FastAPI 專家

> 這些用戶在 [GitHub 上幫助他人解答問題](help-fastapi.md#help-others-with-questions-in-github){.internal-link target=_blank}。 🙇

Author: 已調整

> 他們通過幫助其他人，證明了自己是 **FastAPI 專家**。 ✨

Reviewer: ```suggestion
他們透過幫助其他人，證明了自己是 **FastAPI 專家**。 ✨
```

Author: 已調整

!!! 提示
    你也可以成為官方的 FastAPI 專家！

>     只需要在 [GitHub 上幫助他人解答問題](help-fastapi.md#help-others-with-questions-in-github){.internal-link target=_blank}。 🤓

Reviewer 3: ```suggestion
    只需要在 [幫助其他人解決 GitHub 中的問題](help-fastapi.md#help-others-with-questions-in-github){.internal-link target=_blank}。 🤓
```
The same with above.

Author: 已調整

你可以查看這些期間的 **FastAPI 專家**：

* [上個月](#fastapi-experts-last-month) 🤓
* [過去 3 個月](#fastapi-experts-3-months) 😎
* [過去 6 個月](#fastapi-experts-6-months) 🧐
* [過去 1 年](#fastapi-experts-1-year) 🧑‍🔬
* [**所有時間**](#fastapi-experts-all-time) 🧙

### FastAPI 專家 - 上個月

> 上個月在 [GitHub 上幫助他人解決問題最多的](help-fastapi.md#help-others-with-questions-in-github){.internal-link target=_blank}用戶。 🤓

Reviewer 3: ```suggestion
上個月在 [GitHub 中幫助其他人解決問題最多的](help-fastapi.md#help-others-with-questions-in-github){.internal-link target=_blank} 用戶。 🤓
```
看會不會比較好？

Author: 已調整

{% if people %}
<div class="user-list user-list-center">
{% for user in people.last_month_experts[:10] %}

<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">回答問題數： {{ user.count }}</div></div>
{% endfor %}

</div>
{% endif %}

### FastAPI 專家 - 過去 3 個月

> 過去三個月在 [GitHub 上幫助他人解決問題最多的](help-fastapi.md#help-others-with-questions-in-github){.internal-link target=_blank}用戶。 😎

Reviewer 3: ```suggestion
過去三個月在 [GitHub 中幫助其他人解決問題最多的](help-fastapi.md#help-others-with-questions-in-github){.internal-link target=_blank} 用戶。 😎
```
看會不會比較好？

Author: 已調整

{% if people %}
<div class="user-list user-list-center">
{% for user in people.three_months_experts[:10] %}

<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">回答問題數： {{ user.count }}</div></div>
{% endfor %}

</div>
{% endif %}

### FastAPI 專家 - 過去 6 個月

> 過去六個月在 [GitHub 上幫助他人解決問題最多的](help-fastapi.md#help-others-with-questions-in-github){.internal-link target=_blank}用戶。 🧐

Reviewer 3: ```suggestion
過去六個月在 [GitHub 中幫助其他人解決問題最多的](help-fastapi.md#help-others-with-questions-in-github){.internal-link target=_blank} 用戶。 🧐
```

Author: 已調整

{% if people %}
<div class="user-list user-list-center">
{% for user in people.six_months_experts[:10] %}

<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">回答問題數： {{ user.count }}</div></div>
{% endfor %}

</div>
{% endif %}

### FastAPI 專家 - 過去一年

> 過去一年在 [GitHub 上幫助他人解決問題最多的](help-fastapi.md#help-others-with-questions-in-github){.internal-link target=_blank}用戶。 🧑‍🔬

Reviewer 2: ```suggestion
過去一年在 [GitHub 上幫助他人解決最多問題的](help-fastapi.md#help-others-with-questions-in-github){.internal-link target=_blank}。 🧑‍🔬
```
不曉得這樣有沒有更接近中文表達

Author: 感謝! 我這邊調整成 GitHub "中"幫助他人解決"最多問題的用戶"

```
過去一年在 [GitHub 中幫助他人解決最多問題的](help-fastapi.md#help-others-with-questions-in-github){.internal-link target=_blank}用戶。 🧑‍🔬
```

> 過去一年在 [GitHub 上幫助他人解決問題最多的](help-fastapi.md#help-others-with-questions-in-github){.internal-link target=_blank}用戶。 🧑‍🔬

Reviewer 3: ```suggestion
過去一年在 [GitHub 中幫助其他人解決問題最多的](help-fastapi.md#help-others-with-questions-in-github){.internal-link target=_blank} 用戶。 🧑‍🔬
```

Author: 已調整

{% if people %}
<div class="user-list user-list-center">
{% for user in people.one_year_experts[:20] %}

<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">回答問題數： {{ user.count }}</div></div>
{% endfor %}

</div>
{% endif %}

### FastAPI 專家 - 全部時間

以下是全部時間的 **FastAPI 專家**。 🤓🤯

> 過去在 [GitHub 上幫助他人解決問題最多的](help-fastapi.md#help-others-with-questions-in-github){.internal-link target=_blank}用戶。 🧙

Reviewer 3: ```suggestion
過去在 [GitHub 中幫助其他人解決問題最多的](help-fastapi.md#help-others-with-questions-in-github){.internal-link target=_blank} 用戶。 🧙
```

Author: 已調整

{% if people %}
<div class="user-list user-list-center">
{% for user in people.experts[:50] %}

<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">回答問題數： {{ user.count }}</div></div>
{% endfor %}

</div>
{% endif %}

> ## 核心貢獻者

Reviewer: "top contributors" 的 "top" 不一定是核心貢獻，應該單純指數量而已

Author: "核心"貢獻者好像用字有點太沈重了，我這邊調整成 "主要"貢獻者

還是 Matt 有其他的建議～

Reviewer: i think it's good enough
查到也有人會用「最高貢獻者」([ref](https://community.intel.com/t5/tkb/topcontributorspage?profile.language=zh-TW&countrylabel=Colombia)) 但好像不太常聽到

以下是**核心貢獻者**。 👷

這些用戶[建立了最多已被**合併**的 Pull Requests](help-fastapi.md#create-a-pull-request){.internal-link target=_blank}。

> 他們貢獻了程式碼、文件和翻譯等。 📦

Reviewer: 特別指 source code
```suggestion
他們貢獻了原始碼、文件和翻譯等。 📦
```

Author: 已調整

{% if people %}
<div class="user-list user-list-center">
{% for user in people.top_contributors[:50] %}

<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Pull Requests: {{ user.count }}</div></div>
{% endfor %}

</div>
{% endif %}

還有許多其他的貢獻者（超過一百位），你可以在 <a href="https://github.com/tiangolo/fastapi/graphs/contributors" class="external-link" target="_blank">FastAPI GitHub 貢獻者頁面</a>查看。 👷

## 主要翻譯審核者

以下是 **主要翻譯審核者**。 🕵️

> 我只會講幾種語言（而且不是很流利 😅）。所以審核者[**擁有批准翻譯**](contributing.md#translations){.internal-link target=_blank}文件的權限。沒有他們，就不會有多語言版本的文件。

Reviewer 2: ```suggestion
我只會講幾種語言（而且不是很流利 😅），所以審核者[**擁有批准翻譯**](contributing.md#translations){.internal-link target=_blank}文件的權限。沒有他們，就不會有多語言版本的文件。
```

Author: 已調整

{% if sponsors %}

{% if sponsors.gold %}

### 金牌贊助商

{% for sponsor in sponsors.gold -%}
<a href="{{ sponsor.url }}" target="_blank" title="{{ sponsor.title }}"><img src="{{ sponsor.img }}" style="border-radius:15px"></a>
{% endfor %}
{% endif %}

{% if sponsors.silver %}

### 銀牌贊助商

{% for sponsor in sponsors.silver -%}
<a href="{{ sponsor.url }}" target="_blank" title="{{ sponsor.title }}"><img src="{{ sponsor.img }}" style="border-radius:15px"></a>
{% endfor %}
{% endif %}

{% if sponsors.bronze %}

### 銅牌贊助商

{% for sponsor in sponsors.bronze -%}
<a href="{{ sponsor.url }}" target="_blank" title="{{ sponsor.title }}"><img src="{{ sponsor.img }}" style="border-radius:15px"></a>
{% endfor %}
{% endif %}

{% endif %}

### 個人贊助商

{% if github_sponsors %}
{% for group in github_sponsors.sponsors %}

<div class="user-list user-list-center">

{% for user in group %}
{% if user.login not in sponsors_badge.logins %}

<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a></div>

{% endif %}
{% endfor %}

</div>

{% endfor %}
{% endif %}

## 關於數據 - 技術細節

> 這個頁面的主要目的是突顯社群幫助他人的努力。

Reviewer 3: ```suggestion
這個頁面的主要目的是突顯社群幫助他人所做的努力。
```

Author: 已調整

特別是那些通常不太顯眼但往往更加艱辛的工作，例如幫助他人解答問題和審查包含翻譯的 Pull Requests。

> 這些數據每月計算一次，你可以在查看<a href="https://github.com/tiangolo/fastapi/blob/master/.github/actions/people/app/main.py" class="external-link" target="_blank">原始碼</a>。

Reviewer 3: ```suggestion
這些數據每月計算一次，你可以在這查看<a href="https://github.com/tiangolo/fastapi/blob/master/.github/actions/people/app/main.py" class="external-link" target="_blank">原始碼</a>。
```

> 這些數據每月計算一次，你可以在 <a href="https://github.com/tiangolo/fastapi/blob/master/.github/actions/people/app/main.py" class="external-link" target="_blank">source code 查看</a>。

Reviewer: source code 前面有翻譯
```suggestion
這些數據每月計算一次，你可以在查看<a href="https://github.com/tiangolo/fastapi/blob/master/.github/actions/people/app/main.py" class="external-link" target="_blank">原始碼</a>。
```

Author: 已調整

此外，我也特別表揚贊助者的貢獻。

> 我也保留更新演算法、章節、閾值等的權利（以防萬一 🤷）。

Reviewer 3: ```suggestion
我也保留更新演算法、章節、門檻值等的權利（以防萬一 🤷）。
```
ref: https://hackmd.io/@l10n-tw/glossaries

Author: 已調整

> 我也保留更新算法、章節、閾值等的權利（以防萬一 🤷）。

Reviewer: ```suggestion
我也保留更新演算法、章節、閾值等的權利（以防萬一 🤷）。
```

Author: 已調整
